xbmc/music/tags/MusicInfoTag.cpp

```diff
@@ -582,6 +582,7 @@ void CMusicInfoTag::SetArtist(const CArtist& artist)
   SetArtist(artist.strArtist);
   SetAlbumArtist(artist.strArtist);
   SetMusicBrainzArtistID({ artist.strMusicBrainzArtistID });
+  SetMusicBrainzAlbumArtistID({ artist.strMusicBrainzArtistID });
```

Comment:
Just following what was done previously. When using CMusicInfoTag to hold just artitst data they put that into both artist and albumartist parts of the object. Repetition, but robust too. I think that is OK, but will change it if it upsets anyone.